.ci/lib.sh

```diff
@@ -320,8 +320,10 @@ gen_clean_arch() {
 
 	info "kill stale process"
 	kill_stale_process
-	info "delete stale docker resource under ${stale_docker_dir_union[@]}"
-	delete_stale_docker_resource
+	if [ -n "${USE_PODMAN}" ]; then
```

Comment:
shouldn't this be `[ -z "${USE_PODMAN}" ]`

Reply:
Of course. Updated in some more places.